.env.example

```diff
@@ -15,7 +15,7 @@ DOMAIN=http://localhost:4242
 
 # Supported payment methods for the store.
 # Some payment methods support only a subset of currencies.
-PAYMENT_METHODS="card"
-# PAYMENT_METHODS="card, ideal" # Requires CURRENCY=eur
-# PAYMENT_METHODS="card, fpx" # Requires CURRENCY=myr
+PAYMENT_METHOD_TYPES="card"
+# PAYMENT_METHOD_TYPES="card, ideal" # Requires CURRENCY=eur
```

Comment:
let's also update the comment to have no spaces between `,` and the start of `ideal` that way we don't need to figure out how to trim in all languages :)